src/plugins/home/public/application/components/homepage/section/basics.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCard,
+  EuiIcon,
+  EuiListGroup,
+  EuiListGroupItemProps,
+  IconType,
+} from '@elastic/eui';
+import { Section } from './section';
+
+export const BasicsSection: React.FC<{ initiallyOpen?: boolean }> = ({ initiallyOpen }) => {
+  const gettingStartedLinks: EuiListGroupItemProps[] = [
```

Comment:
should be an integration point. as a plugin i can add my discovery links and we will render